docs/source/clusters.rst

```diff
@@ -23,90 +25,153 @@ Which one you choose is largely dependent on your processes, requirements and ho
 
 Option \(2\) is encouraged because it provides segmentation for ingestion, analysis and storage, on a per-cluster basis, ensuring that folks only have access to the infrastructure and data they need to get their job done.
 
-**Configuration**
+Configuration
+-------------
 
-Open ``variables.json`` and define the ``clusters`` field.  Each key is the name of a cluster, with its value being which AWS region you want to create it in.
+Each cluster lives in its own ``json`` file in the ``conf/clusters`` directory.
 
-Example::
+A cluster file contains name, region, modules, and outputs.
 
-    "clusters": {
-        "corp_laptops": "us-east-1",
-        "corp_servers": "us-east-2",
-        "pci": "us-east-1",
-        "production": "us-west-2"
-    },
-
-Template::
+An example ``production`` cluster::
 
-    {
-      "id": "cluster-name",
-      "region": "region-name",
-      "modules": {
-        "stream_alert": {
-          "alert_processor": {
-            "timeout": 25,
-            "memory": 128,
-            "current_version": "$LATEST",
-            # Optional VPC configuration
-            "vpc_config": {
-              "subnet_ids": [],
-              "security_group_ids": []
-            },
-            # Required for custom S3 buckets and Lambda functions as outputs
-            "outputs": {
-              "aws-s3": [],
-              "aws-lambda": []
-            }
-          },
-          "rule_processor": {
-            "timeout": 10,
-            "memory": 256,
-            "current_version": "$LATEST",
-            # Required if custom SNS inputs as configured
-            "inputs": {
-              "aws-sns:": []
-            }
-          }
-        },
-        "cloudwatch_monitoring": {
-          "enabled": true
+  {
+    "id": "production",
```

Comment:
Is it a requirement that the `id` value matches the cluster file name (ie: `"id": "production"` and file name = `production.json`)? Why is the `id` value necessary?